tests/data/test_metapath_walker.py

```diff
@@ -317,11 +317,11 @@ def test_init_parameters(self):
         assert all(np.array_equal(w1, w2) for w1, w2 in zip(run_1, run_2))
 
     def test_benchmark_uniformrandommetapathwalk(self, benchmark):
-        g = example_graph_random(n_nodes=50, n_edges=500, node_types=2)
+        g = example_graph_random(n_nodes=50, n_edges=2000, node_types=4)
         mrw = UniformRandomMetaPathWalk(g)
 
         # this should be made larger to be more realistic, when it is fast enough
-        nodes = np.arange(0, 5)
+        nodes = np.arange(0, 10)
```

Comment:
This is probably now fast enough to switch to being the same as the other ones, e.g.

https://github.com/stellargraph/stellargraph/blob/284c89d1aaf0471e67d6a6557e7f156b9aa70573/tests/data/test_uniform_random_walker.py#L245

```suggestion
        nodes = np.arange(0, 50)
```